tchannel/tchannel.py

```diff
@@ -132,7 +132,7 @@ def call(self, scheme, service, arg1, arg2=None, arg3=None,
             arg2=arg2,
```

Comment:
BTW have a look at the retry_limit docstring on line 91. It says it's a string. Should be int?

Also, finally, that docstring is actually correct. Before this, it was wrong (since we were doing num_attempts with retry_limit).